core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerPool.scala

```diff
@@ -302,6 +307,33 @@ class ContainerPool(childFactory: ActorRefFactory => ActorRef,
     case RescheduleJob =>
       freePool = freePool - sender()
       busyPool = busyPool - sender()
+    case prewarmConfigList: PreWarmConfigList =>
+      val passedPrewarmConfig = prewarmConfigList.list
+      var newPrewarmConfig: List[PrewarmingConfig] = List.empty
+      latestPrewarmConfig foreach { config =>
+        newPrewarmConfig = newPrewarmConfig :+ passedPrewarmConfig
+          .find(passedConfig =>
+            passedConfig.exec.kind == config.exec.kind && passedConfig.memoryLimit == config.memoryLimit)
+          .getOrElse(config)
+      }
+      latestPrewarmConfig = newPrewarmConfig
```

Comment:
Support just change specify runtime config, e.g.
Let's assume runtimes.json include nodejs:12, python:2, swift4.1, we can change nodejs:12 runtime only via `just pass nodejs:12 runtime info`.